DevelopGitHub/GitHubExtension/GitHubExtension.Tests/Settings.StyleCop

```diff
@@ -0,0 +1,163 @@
+<StyleCopSettings Version="105">
```

Comment:
What do you thinks is it optimal to have StyleCop settings file inside each project?